ocaml/xapi/xapi_vusb.ml

```diff
@@ -53,7 +53,7 @@ let destroy ~__context ~self =
   let r = Db.VUSB.get_record_internal ~__context ~self in
   let vm = r.Db_actions.vUSB_VM in
   (* Force the user to unplug first *)
-  if Helpers.is_running ~__context ~self:vm  && r.Db_actions.vUSB_attached <> Ref.null then
+  if r.Db_actions.vUSB_attached <> Ref.null then
```

Comment:
We usually use `is_valid_ref` rather than a direct comparison with Ref.null

Reply:
updated @jonludlam